### ISSUE TITLE
Sign in with Google temporarily disabled for this app

### ISSUE THREAD

**Issue by mcgyver83** [jdgeenen/gcontactsync#162]
Hi, since a month I had issue with gcontactsync.
I decided to remove and reinstall it and now I have this message:
Sign in with Google temporarily disabled for this app

I tried with 3.1.7 and thunderbird 68.9.0.

Any hints?

**Comment by npalix** [comment]
Got the same issue on a fresh install, trying to use an app password.
It finally works with a normal 2-step authentication and the gContactSync app is now registered
at https://myaccount.google.com/u/1/permissions.

Check your "Third-party apps with account access" in the "Security" section of your Google account.

**Comment by mcgyver83** [comment]
No luck.
gcontactsync 3.1.7
![image](https://user-images.githubusercontent.com/3140780/86886992-30b7dd80-c0f8-11ea-989a-0554612a7e00.png)
despite it has access
![image](https://user-images.githubusercontent.com/3140780/86887046-44fbda80-c0f8-11ea-92d2-1a34c36e8424.png)

**Comment by francwalter** [comment]
I have the same issue since some weeks only.
In the security settings it is allowed since 2019, only now it makes problems.
The Add-On is one reason I stick with TB 68.
But if it really won't work i guess i have to update TB in the end ... 
Edit: I think it could have stopped working already June 2021 because the Google Contacts API stopped working, see: 
https://github.com/jdgeenen/gcontactsync/issues/163#issuecomment-740745571

**Comment by mcgyver83** [comment]
To be honest I did not found a solution.
I moved to a different extension. I think is not nice to write here the link, drop me a message @francwalter if you are interested.

**Comment by francwalter** [comment]
> To be honest I did not found a solution.
> I moved to a different extension. I think is not nice to write here the link, drop me a message @francwalter if you are interested.

@mcgyver83 yes I'm interested! But how could I drop you a message here, there is no way to drop private messages here?

**Comment by mcgyver83** [comment]
Ops.. you are right :(
Here the link for 1 day https://pastebin.com/GUw1SMdr

**Comment by francwalter** [comment]
Ah, OK, I know this extension already :)
It's the normal replacement.
This extension here is dead anyway, so no mystery.
Thank anyway.
